### PR TITLE
Add missing file from cms auth commit

### PR DESCRIPTION
**Pull request by anoopksreyas**
Forgot to add a file with my previous commit related to cms authn.

### CODE REVIEW

File: cms/static/js/factories/login.js [hunk 1]
```diff
@@ -35,7 +39,7 @@ define(['jquery.cookie', 'utility'], function() {
                 } else if($('#login_error').length === 0) {
                     $('#login_form').prepend(
                         '<div id="login_error" class="message message-status error">' +
-                        json.value +
+                        json.value +
```
Comment (rohit-k) [inline cms/static/js/factories/login.js]: trailing whitespace should be removed

File: cms/static/js/factories/login.js [hunk 2]
```diff
@@ -45,6 +49,7 @@ define(['jquery.cookie', 'utility'], function() {
                         .addClass('is-shown')
                         .html(json.value);
                 }
+
```
Comment (rohit-k) [inline cms/static/js/factories/login.js]: remove this extra line

Comment (anoopksreyas) [inline cms/static/js/factories/login.js]: Fixed.

File: cms/static/js/factories/login.js (hunk 1 above)
Comment (anoopksreyas) [inline cms/static/js/factories/login.js]: Removed.

**Comment by rohit-k** [comment]
lgtm

**Pull request merged**